src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -1058,6 +1063,58 @@ def testEmptyFileStoreIDIsReadable(self):
                 self.assertTrue(os.path.isfile(path))
             finally:
                 os.unlink(path)
+        
+        def check_status(self, status, status_fn, seconds=10):
+            i = 0.0
+            while status_fn(self.jobstore_initialized) != status:
+                time.sleep(0.5)
+                i += 0.5
+                if i > seconds:
+                    s = status_fn(self.jobstore_initialized)
+                    self.assertEqual(s, status, 'Status took longer than 10 seconds to fetch:  %s' % s)
+        
+        @travis_test
+        def testGetStatusFailedToilWF(self):
+            """
+            Test that ToilStatus.getStatus() behaves as expected with a failing Toil workflow.
+
+            While this workflow could be called by importing and evoking its main function, doing so would remove the
+            opportunity to test the 'RUNNING' functionality of getStatus().
+            """
+            # --badWorker is set to force failure.
+            wf = subprocess.Popen([python, '-m', 'toil.test.sort.sort',
+                                  'file:' + self.jobstore_initialized.jobStoreDir,
+                                  '--clean=never',
+                                  '--numLines=1', '--lineLength=1', '--badWorker=1'])
+
+            self.check_status('RUNNING', status_fn=ToilStatus.getStatus)
+            wf.wait()
+            self.check_status('ERROR', status_fn=ToilStatus.getStatus)
```

Comment:
What's probably happening here is you're using the same jobstore (`self.jobstore_initialized.jobStoreDir`) for different tests.  In the old tests, it would create a new jobstore with a tmp path for each test to avoid them stepping on each other's feet.

I would keep these in the utilsTest file.  You should be able to write a new test for your new functionality in jobStoreTest without moving these.